editors/code/src/util.ts

```diff
@@ -12,42 +12,42 @@ export function assert(condition: boolean, explanation: string): asserts conditi
     }
 }
 
-export const log = new class {
+export const log = new (class {
```

Comment:
Meh, the rule that requires parentheses for `new` calls...

Reply:
Ah, yeah that one is ugly. I'll try to disable it.